machines-endpoints/pkg/machines-endpoints/main.go

```diff
@@ -418,4 +422,74 @@ func main() {
 			log.ErrorExit(err)
 		}
 	}
+
+	if *flgBMCLogCollectorConfigMap {
+		// create bmc-log-collector configmap of all servers that include boot servers
+		addersses, err := updateServerAddressList(machines)
+		if err != nil {
+			log.ErrorExit(err)
+		}
+		err = client.writeConfigMap(ctx, addersses)
+		if err != nil {
+			log.ErrorExit(err)
+		}
+	}
+}
+
+func updateServerAddressList(machines []Machine) (map[string]string, error) {
```

Comment:
I don't know why `updateServerAddressList()` and `client.writeConfigMap()` are defined after `main()`.

The existing functions are defined in the order of the client's receivers, other functions, and `main()`.
Please check the location where defining them.

Reply:
Moved before main()